ports/servoshell/desktop/minibrowser.rs

```diff
@@ -78,7 +78,7 @@ fn truncate_with_ellipsis(input: &str, max_length: usize) -> String {
 impl Minibrowser {
     pub fn new(
         rendering_context: &RenderingContext,
-        events_loop: &EventsLoop,
+        event_loop: &winit::event_loop::EventLoop<WakerEvent>,
```

Comment:
It's a bit odd to only used a qualified use for one of these types, so I would also do the same for `EventLoop`.

```suggestion
        event_loop: &EventLoop<WakerEvent>,
```